.gitignore

```
@@ -17,6 +17,6 @@ __debug_*
 local.go
 resolver.go
 
-api
+/api
```

I wonder if this will work better ? 
```suggestion
api
!api/
```

---

That *should ignore api, except for any api directory